jmclient/jmclient/maker.py

```diff
@@ -118,7 +118,7 @@ def reject(msg):
         return (True, utxos, auth_pub, cj_addr, change_addr, btc_sig)
 
     @hexbin
-    def on_tx_received(self, nick, tx_from_taker, offerinfo):
+    def on_tx_received(self, nick, tx, offerinfo):
```

Comment:
Curious why did you change the name of this variable? Wasn't the "from_taker" moderately helpful?